web3.js/test/program-tests/vote.test.ts

```diff
@@ -167,6 +167,24 @@ describe('VoteProgram', () => {
 
       // Withdraw from Vote account
       let recipient = Keypair.generate();
+      const voteBalance = await connection.getBalance(newVoteAccount.publicKey);
+
+      expect(
+        VoteProgram.safeWithdraw(
+          {
+            votePubkey: newVoteAccount.publicKey,
+            authorizedWithdrawerPubkey: authorized.publicKey,
+            lamports: voteBalance,
```

Comment:
how about changing this to `voteBalance - minimumAmount + 1` to test the limit? Or you can add another case for it, whichever you prefer